app/javascript/components/App.jsx

```diff
@@ -1,18 +1,86 @@
 import React, { Component } from 'react';
 import StreetDropdown from './intersecting-streets/StreetDropdown';
 import ChildSurveys from './intersecting-streets/ChildSurveys';
-
+import { Button } from 'semantic-ui-react';
+import axios from 'axios';
 import './intersecting-streets/App.css';
 
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
+function App() {
+  const csrfToken = document.querySelector('[name=csrf-token]').content
+
+  const verifySubmission = (event) => {
+    event.preventDefault();
+
+    const reactRoot = document.querySelector('#root');
```

Comment:
What do you think about [inline temp](https://refactoring.guru/inline-temp) refactor for this?